script.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * fn - Variable to store the first name entered by the user.
+ * ln - Variable to store the last name enetered by the user.
+ */
+function getFullName() {
```

Comment:
Please remove those functions which you are not using.

Reply:
Done